### PR TITLE
Ben Tsao

### PR DESCRIPTION
**Pull request by cbtsao47**

### CODE REVIEW

File: Article/Article.js [hunk 1]
```diff
@@ -1,26 +1,32 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
```
Comment (szincone) [inline Article/Article.js]: I love your commit messages ('somehow got expandbutton to work')

File: Article/Article.js [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
```
Comment (szincone) [inline Article/Article.js]: I like the param name change to something that makes more sense to you.

File: Article/Article.js [hunk 3]
```diff
@@ -1,26 +1,32 @@
...
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
+    // console.log(this.domElement);
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
```
Comment (szincone) [inline Article/Article.js]: Could also be written as `this.expandButton.addEventListener("click", () => this.expandArticle());`, no difference in what it does though, just preference.

File: index.html [hunk 4]
```diff
@@ -24,7 +25,7 @@ <h1>Lambda School Newsfeed</h1>
     </div>
     <div class="articles">
       <div class="article">
-        <h2>Lambda School Students: "We're the best!"</h2>
+        <h2>IDK WHAT I'M DOING</h2>
```
Comment (szincone) [inline index.html]: I like that you have fun w/ the project and make it your own.

File: Menu/Menu.js [hunk 5]
```diff
@@ -1,10 +1,21 @@
...
 // Using your menuButton reference, add a click handler that calls toggleMenu
+let onOff = false;
+const toggleMenu = () => {
+  if(onOff == false){
+    onOff = true;
+    menu.classList.add('menu--open');
+    TweenMax.to('.menu', 1, {left:-43, opacity: 1 ,ease:Power4.easeOut});  
+  }else if (onOff == true){
+    onOff = false;
+    TweenMax.to('.menu', 1, {left:-350, opacity: 0, ease:Power4.easeOut, onComplete: () => menu.classList.remove('menu--open')});
```
Comment (szincone) [inline Menu/Menu.js]: I can really see you went into the greensock docs to find out about the methods.

File: index.html (hunk 4 above)
Comment (cbtsao47) [reply index.html]: lol thanks man

File: Article/Article.js (hunk 3 above)
Comment (cbtsao47) [reply Article/Article.js]: Ahh, I forgot about that one. thanks for pointing that out. Gotta get used to arrow functions !!

**Pull request closed without merging**